apps/OboeTester/docs/AutomatedTesting.md

```diff
@@ -58,7 +58,7 @@ For example:
 
 ## Parameters
 
-There is one required parameter for all tests:
+There are two required parameters for all tests:
```

Comment:
That will break vendor's existing tests scripts.
For Input and Output test allow no report file.

Reply:
This will not break anything. Having a report log is optional for all tests when you run the tests. We are just saying "required" here to encourage everyone to use one.

Reply:
I thought the file was required but the test seems to run without it!
That is weird but the problem is not new to this CL. OK.